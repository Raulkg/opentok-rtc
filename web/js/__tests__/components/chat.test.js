var Chat = require('../../components/chat.js');
test('snapshot for Chat component', () => {
  expect(Chat).toMatchSnapshot();
});