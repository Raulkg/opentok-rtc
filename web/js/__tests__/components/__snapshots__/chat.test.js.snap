// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for Chat component 1`] = `
Object {
  "Chat": Object {
    "collapse": [Function],
    "expand": [Function],
    "hide": [Function],
    "init": [Function],
    "isCollapsed": [Function],
    "show": [Function],
    "visible": null,
  },
}
`;
